Formula/node.rb

```diff
@@ -107,12 +89,18 @@ def post_install
       # Dirs must exist first: https://github.com/Homebrew/legacy-homebrew/issues/35969
       mkdir_p HOMEBREW_PREFIX/"share/man/#{man}"
       rm_f Dir[HOMEBREW_PREFIX/"share/man/#{man}/{npm.,npm-,npmrc.,package.json.}*"]
-      ln_sf Dir[libexec/"npm/lib/node_modules/npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
+      ln_sf Dir[libexec/"npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
     end
 
     npm_root = node_modules/"npm"
     npmrc = npm_root/"npmrc"
     npmrc.atomic_write("prefix = #{HOMEBREW_PREFIX}\n")
+
+    # This is a minor hack that let's us keep a working npm
+    # in both HOMEBREW_PREFIX & also privately in libexec.
```

Comment:
As I said here https://github.com/Homebrew/homebrew-core/pull/14085#discussion_r119527275 it seems to me that packing-before-installing should at least be an option in upstream's configure. In the absence of such an option, doing it ourselves makes sense if it works.